gok8s/workload/workload.go

```diff
@@ -80,17 +80,19 @@ func CheckPodSpecVersion(podSpec corev1.PodSpec, kcd *kcdv1.KCD, versions ...str
 	for _, c := range podSpec.Containers {
 		if c.Name == kcd.Spec.Container.Name {
 			match = true
-			parts := strings.SplitN(c.Image, ":", 2)
-			if len(parts) > 2 {
+			r := regexp.MustCompile(`(.*)\:(.*)`)
+			parts := r.FindStringSubmatch(c.Image)
```

Comment:
this only matches something like `localhost:8081/wish/wish-fe:abcdefg`, what if I still need `888888888888.dkr.ecr.us-west-1.amazonaws.com/wish/kcd:abcdefg`?

Reply:
it should work in both cases